src/test/resources/expected/lambda.kore.golden

```diff
@@ -1,692 +1 @@
-                                                            , symbolOrAliasParams = []
-                                                            }
-                                                    , applicationChildren =
-                                                        [ MetaPattern (VariablePattern Variable
-                                                            { variableName = Id Meta "#E"
-                                                            , variableSort =
-                                                                SortActualSortSortActual
-                                                                        { sortActualName = Id Meta "#Pattern"
-                                                                        , sortActualSorts = []
-                                                                        }
-                                                            })
-                                                        , MetaPattern (VariablePattern Variable
-                                                            { variableName = Id Meta "#E'"
-                                                            , variableSort =
-                                                                SortActualSortSortActual
-                                                                        { sortActualName = Id Meta "#Pattern"
-                                                                        , sortActualSorts = []
-                                                                        }
-                                                            })
-                                                        , ObjectPattern (VariablePattern Variable
-                                                            { variableName = Id Object "X"
-                                                            , variableSort =
-                                                                SortActualSortSortActual
-                                                                        { sortActualName = Id Object "Exp"
-                                                                        , sortActualSorts = []
-                                                                        }
-                                                            })
-                                                        ]
-                                                    })
-                                            })
-                                        ]
-                                    })
-                            })
-                    , sentenceAxiomAttributes = Attributes []
-                    }
-                ]
-            , moduleAttributes = Attributes []
-            }
-        ]
-    }
\ No newline at end of file
+Parse error: Error in module 'LAMBDA' -> axiom declaration -> \equals -> symbol or alias '#isExp' -> symbol or alias 'lambda': Expected 2 operands, but got 1..
```

Comment:
I'm not sure this test was supposed to fail... maybe the original .kore file needs to be changed? @hreada

Reply:
@traiansf @virgil-serbanuta There seems to be a mismatch of parentheses in `lambda.kore`. I've made a commit to `master` that fixes the problem. That fix should make this failing test pass.

Reply:
@hreada Thanks, that seems to have fixed something, but the next error is:
Parse error: Error in module 'LAMBDA' -> axiom declaration -> \implies -> symbol or alias '#provable' -> \equals -> symbol or alias '#substitute' -> variable 'X': Expecting meta sort '#Variable{}' but got object sort 'Exp{}'.